examples/Remarketing/AddRemarketingAction.php

```diff
@@ -0,0 +1,192 @@
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     */
+    public static function runExample(GoogleAdsClient $googleAdsClient, int $customerId)
+    {
+        // Creates a remarketing action with the specified name.
+        $remarketingAction = new RemarketingAction([
+            'name' => new StringValue(['value' => 'Remarketing action #' . uniqid()])
+        ]);
+
+        // Creates a remarketing action operation.
+        $remarketingActionOperation =
+            new RemarketingActionOperation(['create' => $remarketingAction]);
+
+        // Issues a mutate request to add the remarketing action and print out some information.
+        $remarketingActionServiceClient = $googleAdsClient->getRemarketingActionServiceClient();
+        $response = $remarketingActionServiceClient->mutateRemarketingActions(
+            $customerId,
+            [$remarketingActionOperation]
+        );
+        $remarketingActionResourceName = $response->getResults()[0]->getResourceName();
+        printf(
+            "Added remarketing action with resource name '%s'.%s",
+            $remarketingActionResourceName,
+            PHP_EOL
+        );
+
+        // Creates a query that retrieves the previously created remarketing action with its
+        // generated tag snippets.
+        $query =
+            "SELECT remarketing_action.id, "
+            . "remarketing_action.name, "
+            . "remarketing_action.tag_snippets "
+            . "FROM remarketing_action "
+            . "WHERE remarketing_action.resource_name = '$remarketingActionResourceName'";
+
+        // Issues a search request by specifying page size.
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        $response =
+            $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
+
+        // There is only one row because we limited the search using the resource name, which is
+        // unique.
+        /** @var GoogleAdsRow $googleAdsRow */
+        $googleAdsRow = $response->iterateAllElements()->current();
+
+        // Print some attributes of the remarketing action. The ID and tag snippets are generated
+        // by the API.
+        printf(
+            "Remarketing action has ID %d and name '%s'.%s%s",
+            $googleAdsRow->getRemarketingAction()->getIdUnwrapped(),
+            $googleAdsRow->getRemarketingAction()->getNameUnwrapped(),
+            PHP_EOL,
+            PHP_EOL
+        );
+        print 'It has the following generated tag snippets:' . PHP_EOL;
+        foreach ($googleAdsRow->getRemarketingAction()->getTagSnippets() as $tagSnippet) {
+            /** @var TagSnippet $tagSnippet */
+            printf(
```

Comment:
nit: add tabulation or bullet to delimitate each tag in the list

Reply:
Thanks for suggestion, but the tags are quite lengthy and hard to read already.
Adding tabulation or bullet will not make it more readable and could confuse some users that those texts are part of the tags.